test_utilities/src/common.rs

```diff
@@ -1135,10 +1136,11 @@ pub fn test_db(
         Kind::Aws => (AWS_TEST_REGION.to_string(), AWS_KUBERNETES_VERSION.to_string()),
         Kind::Do => (DO_TEST_REGION.to_string(), DO_KUBERNETES_VERSION.to_string()),
         Kind::Scw => (SCW_TEST_ZONE.to_string(), SCW_KUBERNETES_VERSION.to_string()),
+        Kind::Edge(Edge::Aws) => (AWS_TEST_REGION.to_string(), AWS_K3S_VERSION.to_string()),
```

Comment:
Can you flatten the kind to `AWS_EDGE` instead of nesting 2 types inside.
Even if we extend that to other providers, there is little chances that we want to first match on Edge before willing to know which provider it is

Reply:
I've redefined this part. I think `Qovery Edge` is a misleading name and does not reflect the goal of this feature (deploying Qovery on an AWS EC2 instance). What I propose is to just create a new Kubernetes provider named `EC2` implementing the `Kubernetes` trait. So we do not need to touch anything. We just need to implement EC2 as a Kubernetes type. Then, as a user, you can choose between EKS and EC2 while creating a cluster on Qovery.